nas_ports/freenas/py-middlewared/files/middlewared.in

```diff
@@ -30,11 +30,10 @@ middlewared_start() {
 	else
 		env PATH=$PATH:/usr/local/sbin:/usr/local/bin LC_ALL=en_US.UTF-8 ${command} -f -P ${pidfile} -r /usr/local/bin/middlewared ${overlay_dirs_arg} --log-handler=file
 	fi
-	if ! LD_LIBRARY_PATH=/usr/local/lib /usr/local/bin/midclt -t 240 waitready; then
-		echo "#################"
-		echo "MIDDLEWARED FAILED TO START WITHIN 4 MINUTES, SYSTEM WILL NOT BEHAVE" \
-		     "CORRECTLY!!"
-		echo "#################"
+	if ! LD_LIBRARY_PATH=/usr/local/lib /usr/local/bin/midclt -t 60 waitready; then
```

Comment:
It is unclear how long one step can take. It could very well be the first plugin loads the majority of modules and all byte-code is compiled there.

Its also possible the startup of middlewared itself takes some time since its a python process and modules will be recompiled before all step writing can take place.

I am not sure decreasing the time here will make this behave any better.